### PR TITLE
Catalog performance  optimization

### PR DESCRIPTION
**Pull request by tatarincev**

### CODE REVIEW

**Comment by lnetrebskii** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-2637 has been created

File: src/VirtoCommerce.CatalogModule.Core/Extensions/EnumerableExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+using System;
```
Comment (yecli) [inline src/VirtoCommerce.CatalogModule.Core/Extensions/EnumerableExtensions.cs:1]: The same method was added into the platform. Maybe need to remove it from here.

Comment (akak1977) [reply src/VirtoCommerce.CatalogModule.Core/Extensions/EnumerableExtensions.cs:1]: Yes we can. But we need to raise the platform dependency to an unreleased version.

Comment (yecli) [reply src/VirtoCommerce.CatalogModule.Core/Extensions/EnumerableExtensions.cs:1]: Or release the platform first, and then use it here.

**Pull request merged**